src/zulip.rs

```diff
@@ -520,3 +568,21 @@ async fn move_notification(
         .unwrap()),
     }
 }
+
+fn message_channel_link(msg: &Message) -> String {
```

Comment:
Can we at least split this addition to a separate commit? I'm also not clear on the use case--Zulip makes it pretty easy to get links, no?

Reply:
Here were my thoughts on this (from this PR's comment):

> The `link` command is meant as a helper for e.g. mobile clients which have no easy way to determine the link to a topic. Its implementation could also be reused in the context of other commands.

Reply:
Removed it and will add it back later